File: decompiled/sources/com/streamax/client/ui/devlist/DevInfoEvent.java
package com.streamax.client.ui.devlist;

public class DevInfoEvent {
    public String mGroupName;
    public int mId;
    public boolean mSingle;

    public DevInfoEvent() {
    }

    public DevInfoEvent(int i, String str) {
        this.mGroupName = str;
        this.mId = i;
    }
}

File: decompiled/sources/com/streamax/client/ui/devlist/DevInfoEvent.kt
package com.streamax.client.ui.devlist

class DevInfoEvent {
    @JvmField var mGroupName: String? = null
    @JvmField var mId: Int = 0
    @JvmField var mSingle: Boolean = false

    constructor()
    constructor(id: Int, groupName: String?) {
        mGroupName = groupName
        mId = id
    }
}
